WORKSPACE

```diff
@@ -1,52 +1,63 @@
 workspace(name = "angular_devkit")
 
+# We get Buildifier from here.
+http_archive(
+    name = "com_github_bazelbuild_buildtools",
+    url = "https://github.com/bazelbuild/buildtools/archive/0.15.0.zip",
+    strip_prefix = "buildtools-0.15.0",
+    sha256 = "76d1837a86fa6ef5b4a07438f8489f00bfa1b841e5643b618e01232ba884b1fe",
+)
+
+load("@com_github_bazelbuild_buildtools//buildifier:deps.bzl", "buildifier_dependencies")
+buildifier_dependencies()
+
+# The Go toolchain is used for Buildifier.
+# rules_typescript_dependencies() also tries to load it but we add it explicitely so we
```

Comment:
It's just for buildifier really. It's not used for anything else. But I had to play around with the order of things because of all the `*_dependencies()` calls. I think the foolproof way is to declare all the `http_archive` (and other repositories) before loading imports.